tests/integration/MediaWiki/Api/EditEndpointTest.php

```diff
@@ -26,7 +26,9 @@ class EditEndpointTest extends \MediaWikiIntegrationTestCase {
 	use HandlerTestTrait;
 
 	private const URL_PROPERTY = 'P1';
+	private const STRING_PROPERTY = 'P2';
 	private const MISSING_PROPERTY = 'P1000';
+	private const QUALIFIER_PROPERTY = 'P666';
```

Comment:
I find this confusing – qualifiers use the same properties in Wikibase as everything else, and “qualifier” is not a datatype in the same way that “url” and “string” are. I suggest removing this, and using `STRING_PROPERTY` in the tests below.

Reply:
I thought using the String property would make other tests fail because of the datatype is not URL. But the tests are passing.

Reply:
I thought some of the tests would fail if I compare them to a property with a datatype that's not URL. But they are passing. Done!